build_r.R

```diff
@@ -75,7 +75,7 @@ version <- gsub(
 )
 tarball <- file.path(getwd(), sprintf("lightgbm_%s.tar.gz", version))
 
-cmd <- sprintf("R CMD INSTALL %s --no-multiarch", tarball)
+cmd <- sprintf("R CMD INSTALL %s --no-multiarch --install-tests --with-keep.source --with-keep.parse.data", tarball)
```

Comment:
@jameslamb My R knowledge is quite poor, so I removed a review request from me. Sorry.

I'm just worrying about `--install-tests`. What is it? Will it increase the installation time significantly, which may cause `RTD` builds to fail due to time limits?

Reply:
good point @StrikerRUS . I actually think I can refactor this to make that unnecessary. Those flags do increase installation time (although it should be very small), but actually we don't need them for this at all. They're passed at build time by `covr::package_coverage()` but since I'm using a super-LightGBM-specific mutation of `covr::package_coverage()`, I can just manually handle copying the test files.

Reply:
done!

Reply:
Thanks!